core/src/main/scala/cromwell/util/GracefulShutdownHelper.scala

```diff
@@ -0,0 +1,34 @@
+package cromwell.util
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Terminated}
+import akka.pattern.GracefulStopSupport
+import cats.data.NonEmptyList
+import cromwell.util.GracefulShutdownHelper.ShutdownCommand
+
+object GracefulShutdownHelper {
+  case object ShutdownCommand
+}
+
+trait GracefulShutdownHelper extends GracefulStopSupport { this: Actor with ActorLogging =>
+  private [util] var shuttingDown: Boolean = false
```

Comment:
Can you make it private-private instead of package private then?

And yeah, it probably could :)